wagtail/admin/tests/test_workflows.py

```diff
@@ -2422,116 +2440,10 @@ def get_file_content(self, response, format):
             )
         return response.getvalue().decode()
 
-    def test_workflow_report(self):
+    def test_workflow_report_not_shown_to_moderators(self):
         response = self.client.get(reverse("wagtailadmin_reports:workflow"))
-        self.assertEqual(response.status_code, 200)
-        self.assertContains(response, "Hello world!")
-        self.assertContains(response, "test_workflow")
-        self.assertContains(response, "Sebastian Mitter")
-        self.assertContains(response, "March 31, 2020")
-
-        response = self.client.get(reverse("wagtailadmin_reports:workflow_tasks"))
-        self.assertEqual(response.status_code, 200)
-        self.assertContains(response, "Hello world!")
-
-    def test_workflow_report_filtered(self):
```

Comment:
I am concerned we are removing all these tests, was this intentional?

Reply:
Yes, the removal of those specific tests from TestPageWorkflowReport was intentional. As the role of the moderator no longer has access to WorkflowReport in the updated code, those tests were no longer relevant.

Also, please let me know if anything appears incorrect or if there's anything you would like me to adjust.

Reply:
I think we would want to still have these tests but maybe update them to use a different user that has the right permissions in the tests. 

Otherwise we are going to be missing a whole lot of critical functionality being tested for these reports.

Reply:
> As the role of the moderator no longer has access to WorkflowReport in the updated code, those tests were no longer relevant.

Is this really what we want? The original issue #8845 was about users _with no page permissions at all_ seeing the report. Users who have some involvement in the page publishing process _should_ be able to see this report, I think?

Reply:
Correct, `"change"` should be used instead of `"edit"` for permissions.